packages/workbox-precaching/test/sw/failing-cache-tests.js

```diff
@@ -20,30 +38,28 @@ describe('Test Failing Cache Behavior', function() {
     });
   });
 
-  it.skip('should fail to install unrevisioned with 404 cache request', function() {
-    const unrevisionedCacheManager = new workbox.precaching.UnrevisionedCacheManager();
-    unrevisionedCacheManager.addToCacheList({
-      unrevisionedFiles: [
-        '/__test/404/',
-      ],
-    });
-    return unrevisionedCacheManager.install()
-    .then(() => {
-      throw new Error('Expected error to be thrown.');
-    }, (err) => {
-      expect(err).to.exist;
-      expect(err.name).to.equal('request-not-cached');
-      expect(err.extras.url).to.equal(new URL('/__test/404/', self.location).toString());
-    });
-  });
+  // it('should fail to install unrevisioned with 404 cache request', () => {
```

Comment:
IIRC, the `UnrevisionedCacheManager` file didn't build, so even just having the reference there was problematic.

I suppose I could leave it unchanged if you think that's preferable.